src/openrct2/ride/TrackDesign.cpp

```diff
@@ -2000,7 +2001,7 @@ static bool track_design_place_preview(TrackDesign* td6, money32* cost, Ride** o
 void track_design_draw_preview(TrackDesign* td6, uint8_t* pixels)
 {
     // Make a copy of the map
-    map_backup* mapBackup = track_design_preview_backup_map();
+    auto mapBackup = track_design_preview_backup_map().get();
```

Comment:
this should not be a `get()` you want this to be the smart pointer

Reply:
L2021 and L2104 will either need to have a `get()` added or use std::move to move the smart pointer into that function and adjusting the function. I suggest just using get as it is safe to do in this context.